webstruct/model.py

```diff
@@ -21,29 +20,30 @@ class NER(object):
 
     Initialize it with a trained ``model``. ``model`` must have
     ``predict`` method that accepts lists of :class:`~.HtmlToken`
-    sequences and returns lists of predicted IOB2 tags.
+    sequences and returns lists of predicted IOB2 or BILOU tags if bilou = True.
     :func:`~.create_wapiti_pipeline` function returns such model.
     """
     HEADERS = {
         'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; WOW64; rv:40.0) Gecko/20100101 Firefox/40.1',
     }
 
     def __init__(self, model, loader=None, html_tokenizer=None,
-                 entity_colors=None):
+                 entity_colors=None, bilou=False):
         self.model = model
         self.loader = loader or HtmlLoader()
-        self.html_tokenizer = html_tokenizer or HtmlTokenizer()
+        self.html_tokenizer = html_tokenizer or HtmlTokenizer(bilou=bilou)
         if entity_colors is None:
             entity_colors = EntityColors()
         self.entity_colors = entity_colors
+        self.bilou = bilou
 
     def extract(self, bytes_data):
         """
         Extract named entities from binary HTML data ``bytes_data``.
         Return a list of ``(entity_text, entity_type)`` tuples.
         """
         html_tokens, tags = self.extract_raw(bytes_data)
-        groups = IobEncoder.group(zip(html_tokens, tags))
+        groups = self.html_tokenizer.sequence_encoder.group(zip(html_tokens, tags))
```

Comment:
It doesn't look correct for bilou html_tokenizers. This will be fixed if we move bilou handling logic to BilouEncoder.